packages/ui/src/lib/components/MoveToPageMenu.tsx

```diff
@@ -10,6 +11,7 @@ export const MoveToPageMenu = track(function MoveToPageMenu() {
 	const pages = app.pages
 	const currentPageId = app.currentPageId
 	const msg = useTranslation()
+	const { addToast } = useToasts()
```

Comment:
Why are we adding toasts in this PR? It's also missing translations.

Reply:
This PR removes the `move-to-page` event and moves that logic from out of `useAppEvents` to the callback here.